client/src/pages/AddTour.js

```diff
@@ -32,6 +33,32 @@ export default function AddTour() {
 
   const concurrentTour = query.get("type") === "concurrent";
 
+  useEffect(() => {
+    if (id) {
+      const doFetch = async () => {
+        try {
+          const data = await getDataByID({
```

Comment:
The naming is very unspecific.
What about this?
```suggestion
          const task = await getTaskById(id);
```

Reply:
Was trying to keep the naming that unspecific so i can keep the functions multifunctional and don't have to write new functions everytime. That way i can get Tasks, Riders and Companies from the according collection with one and the same function. Too lazy to write more functions than needed. 😄